contracts/eosio.system/src/powerup.cpp

```diff
@@ -388,6 +387,10 @@ void system_contract::rentbw(const name& payer, const name& receiver, uint32_t d
    adjust_resources(get_self(), reserv_account, core_symbol, net_delta_available, cpu_delta_available, true);
    channel_to_rex(payer, fee, true);
    state_sing.set(state, get_self());
+
+   // inline noop action
+   powup_results::powupresult_action powupresult_act{ reserv_account, std::vector<eosio::permission_level>{ } };
+   powupresult_act.send( fee, asset{ net_amount, core_symbol }, asset{ cpu_amount, core_symbol } );
```

Comment:
The asset type doesn't make any sense here since powerup severs any link which remained between weights and tokens. It'd be nice to add the `powerup` args to simplify history tools.

Reply:
I've updated the powupresult to return the `net_weight` and `cpu_weight`.